lucene/core/src/java/org/apache/lucene/codecs/lucene90/Lucene90VectorFormat.java

```diff
@@ -33,15 +34,16 @@
 
   static final String META_CODEC_NAME = "Lucene90VectorFormatMeta";
   static final String VECTOR_DATA_CODEC_NAME = "Lucene90VectorFormatData";
-
+  static final String VECTOR_INDEX_CODEC_NAME = "Lucene90VectorFormatIndex";
   static final String META_EXTENSION = "vem";
   static final String VECTOR_DATA_EXTENSION = "vec";
+  static final String VECTOR_INDEX_EXTENSION = "vex";
 
   static final int VERSION_START = 0;
   static final int VERSION_CURRENT = VERSION_START;
 
   /** Sole constructor */
-  public Lucene90VectorFormat() {
+  Lucene90VectorFormat() {
```

Comment:
Hmm, why no longer `public`?  Users making their own Codec might want to use this constructor I think?